locales/languages/en.json

```
@@ -595,7 +596,21 @@
     "decimals_is_required": "Decimal is required. Find it on:",
     "no_tokens_found": "We couldn't find any tokens with that name.",
     "select_token": "Select Token",
-    "address_must_be_smart_contract": "Personal address detected. Enter the token contract address."
+    "address_must_be_smart_contract": "Personal address detected. Enter the token contract address.",
+    "billionAbbreviation": "B",
+    "trillionAbbreviation": "T",
+    "millionAbbreviation": "M",
+    "token_details": "Token details",
+    "contract_address": "Contract address",
+    "token_list": "Token list",
+    "market_details": "Market Details",
```

```suggestion
    "market_details": "Market details",
```

match casing of the other titles